composer.json

```diff
@@ -31,7 +31,8 @@
         "php-mock/php-mock": "^1.0",
         "predis/predis": "^0.8 || ^1.0",
         "sllh/php-cs-fixer-styleci-bridge": "^2.0",
-        "symfony/phpunit-bridge": "^3.3"
+        "symfony/phpunit-bridge": "^3.3",
+        "twig/twig": "^1.3"
```

Comment:
Should be v2 here, or I misunderstood your PR?

Reply:
Sorry, my mistake!